### ISSUE TITLE
Confused about link searching behaviour

### ISSUE THREAD

**Issue by archiif** [alphapapa/helm-org-rifle#56]
From the documentation:
> When =helm-org-rifle-show-path= is enabled, replace Org links in headings with their descriptions.  This prevents =org-format-outline-path= from truncating the links, making them useless for reading.

I tend to have a lot of headings that contains only links with no paragraphs underneath it, and I usually prefer to search with the outline path visible. But the behavior mentioned in the documentation above makes it hard for me to search them because the links are now invisible in the helm buffer. This is especially true when I'm searching based on tags.

Are there any workarounds to this?

**Comment by alphapapa** [comment]
Do you mean that the links in question do not have a description?

**Comment by archiif** [comment]
Yes the description of the links are not visible at all. See this screenshot:

![screenshot](https://i.imgur.com/PVMb9vG.png)

**Comment by alphapapa** [comment]
I just tested this in a clean Emacs config and it works for me:

![wfm](https://user-images.githubusercontent.com/601365/91104641-c9b4a080-e633-11ea-8b3f-52f4042f3e72.png)

**Comment by archiif** [comment]
I guess this might be a Doom-specific issue, I'll look into this further
